projects/1.16/assets/prehistoric-fauna/prehistoricfauna/lang/zh_cn.json

```
@@ -0,0 +1,968 @@
+{
+	"block.prehistoricfauna.osmundacaulis": "紫萁茎",
+	"block.prehistoricfauna.dead_osmundacaulis": "枯萎紫萁茎",
+	"block.prehistoricfauna.dicroidium": "叉蕨",
+	"block.prehistoricfauna.johnstonia": "咀签",
+	"block.prehistoricfauna.cladophlebis": "枝脉蕨",
+	"block.prehistoricfauna.scytophyllum": "革叶",
+	"block.prehistoricfauna.michelilloa": "含笑木兰",
+	"block.prehistoricfauna.horsetail": "马尾草",
+	"block.prehistoricfauna.tall_horsetail": "高马尾草",
+	"block.prehistoricfauna.osmunda": "紫萁",
+	"block.prehistoricfauna.tall_osmunda": "高紫萁",
+	"block.prehistoricfauna.clubmoss": "石松",
+	"block.prehistoricfauna.marchantia": "地钱",
+	"block.prehistoricfauna.moss_carpet": "苔藓地毯",
+	"block.prehistoricfauna.thatch_carpet": "覆草地毯",
+	"block.prehistoricfauna.loam": "壤土",
+	"block.prehistoricfauna.packed_loam": "紧实壤土",
+	"block.prehistoricfauna.moss_block": "苔藓块",
+	"block.prehistoricfauna.coniopteris": "伞形蕨",
+	"block.prehistoricfauna.tall_osmundacaulis": "高紫萁",
+	"block.prehistoricfauna.mossy_dirt": "苔土",
+	"block.prehistoricfauna.algae_carpet": "海藻地毯",
+	"block.prehistoricfauna.silt": "粉土",
+	"block.prehistoricfauna.hardened_silt": "硬化粉土",
+	
+	"block.prehistoricfauna.liriodendrites_planks": "鹅掌楸木板",
+	"block.prehistoricfauna.heidiphyllum_planks": "海蒂菲兰木板",
+	"block.prehistoricfauna.protojuniperoxylon_planks": "原始刺柏木板",
+	"block.prehistoricfauna.zamites_planks": "拟查米亚木板",
+	"block.prehistoricfauna.protopiceoxylon_planks": "原始云杉木板",
+	"block.prehistoricfauna.cypress_planks": "落羽松木板",
+	"block.prehistoricfauna.araucaria_planks" :"南洋杉木板",
+	"block.prehistoricfauna.metasequoia_planks": "水杉木板",
+	"block.prehistoricfauna.ginkgo_planks": "银杏木板",
+	
+	"block.prehistoricfauna.araucaria_sign": "南洋杉告示牌",
+	"block.prehistoricfauna.araucaria_wall_sign": "南洋杉告示牌",
+	"block.prehistoricfauna.metasequoia_sign": "水杉告示牌",
+	"block.prehistoricfauna.metasequoia_wall_sign": "水杉告示牌",
```

```suggestion
	"block.prehistoricfauna.metasequoia_sign": "水杉木告示牌",
	"block.prehistoricfauna.metasequoia_wall_sign": "水杉木告示牌",
```
参考原版云杉是“云杉木告示牌”。
真折磨人（